README.md

```diff
@@ -132,51 +132,90 @@ Run:
 
 #### Mac OS X and iOS
 
-* Download this version of Graal VM: https://download2.gluonhq.com/substrate/graalvm/graalvm-svm-darwin-20.1.0-ea+28.zip and unpack it like you would any other JDK. (e.g. in `/opt`)
+* Download the latest release version of GraalVM: https://github.com/graalvm/graalvm-ce-builds/releases/tag/vm-20.2.0 by choosing `graalvm-ce-java11-darwin-amd64-20.2.0.tar.gz` from the list of assets and unpack it to a preferred location on your system (e.g. in `/opt`)
 
 * Configure the runtime environment. Set `GRAALVM_HOME` environment variable to the GraalVM installation directory.
 
-For example:
+  For example:
 
-    export GRAALVM_HOME=/opt/graalvm-svm-darwin-20.1.0-ea+28
+      export GRAALVM_HOME=/opt/graalvm-ce-java11-20.2.0/Contents/Home
 
-* Set `JAVA_HOME` to point to the GraalVM installation directory
+* Set `JAVA_HOME` to point to the GraalVM installation directory:
 
-For example:
+      export JAVA_HOME=$GRAALVM_HOME
 
-    export JAVA_HOME=$GRAALVM_HOME
+##### Additional requirements
 
-By default `target` is set to `host`. To deploy to iOS, set the target:
+* iOS can be built only on Mac OS X
 
-```
-gluonClient {
-     target = "ios"
- }
-```
+* Xcode 11+ is required to build for iOS 13+. Install `Xcode` from the [Mac App Store](https://apps.apple.com/us/app/xcode/id497799835?mt=12) if you haven't already. 
```

Comment:
Should this be XCode 11.x ? AFAIK, XCode 12 is still not supported.

Reply:
Xcode is supported, though it causes an issue linking for iOS that should be fixed asap.